src/main/java/no/sikt/nva/validators/DublinCoreValidator.java

```diff
@@ -70,9 +74,41 @@ public static Set<ErrorDetails> getDublinCoreErrors(DublinCore dublinCore, Strin
         getMultipleUnmappableTypeError(dublinCore, customer).ifPresent(errors::add);
         getMultipleValues(dublinCore).ifPresent(errors::addAll);
         getLicenseError(dublinCore).ifPresent(errors::add);
+        getProjectError(dublinCore, fundingSources).ifPresent(errors::add);
         return errors;
     }
 
+    private static Optional<ErrorDetails> getProjectError(DublinCore dublinCore, FundingSources fundingSources) {
+        if (!hasProject(dublinCore)) {
```

Comment:
Single-usage private method negated? `doesNotHaveProject` or somesuch

Reply:
Might even have been tempted to pull all of the Project-error related stuff into their own validator that could then be called here